scripts/setup_test.py

```diff
@@ -373,7 +373,7 @@ def test_package_install_with_windows_x86(self):
         def mock_exists(unused_path):
             return False
 
-        def mock_url_retrieve(url, unused_filename):
+        def mock_url_retrieve(url, filename): # pylint: disable=unused-argument
```

Comment:
Oh wait -- why are we changing unused_filename to filename?

Please revert this. Ditto below

Reply:
Because in python_utils file I changed filename to a keyword argument and now we need to use its original name every where

Reply:
Can you please revert this? This has to be unused_filename since it makes more sense.